db/migrate/20150316135303_create_course_enrol_requests.rb

```diff
@@ -0,0 +1,15 @@
+class CreateCourseEnrolRequests < ActiveRecord::Migration
+  def change
+    create_table :course_enrol_requests do |t|
+      t.belongs_to :user
```

Comment:
should put a <code>null: false </code> for all the columns which aren't supposed to be null.